docs/testbed/sai_quality/PTF-SAIv2TestingGuide.md

```diff
@@ -0,0 +1,137 @@
+# PTF-SAIv2 testing guide
+## Overview
+This article will introduce the detailed processing of entire PTF-SAIv2 testing, including:
+* Build PTF-SAIv2 infras leveraged by [sonic-buildimage](https://github.com/Azure/sonic-buildimage)
+* Setup the testbed by sonic-mgmt
+* Setup saiserverv2 docker on DUT (Device under testing)
+* Prepare the testing env and start PTF-SAIv2 testing within ptf-sai docker
+
+## Build PTF-SAIv2 infras leveraged by sonic-buildimage
+In this part, we will build PTF-SAIv2 infras using sonic-buildimage.
+
+1. Check the sonic image version and commit id: [Check SAI Header Version And SONiC Branch](https://github.com/Azure/sonic-mgmt/blob/master/docs/testbed/sai_quality/ExampleCheckSonicVersionAndBuildSaiserverDocker.md)
+2. Reset the sonic-buildimage with the branch and commit id previous checked
+    ```
+    rm -rf ./sonic-buildimage
+    git clone https://github.com/Azure/sonic-buildimage.git
+    cd sonic-buildimage
+
+    git checkout <specific branch>
+    git reset --hard <specific commit id>
+    ```
+3. Build PTF-SAIv2 infras 
+    ```
+    # Init env
+    make init
+    # BLDENV=buster: Current image is buster
+    # SAITHRIFT_V2=y: build the saiserver version 2rd
+    # Setup platform environment e.g. broadcom
+    make BLDENV=buster configure PLATFORM=broadcom
```

Comment:
this docker is taken brcm as an example? is there any configuration difference on other platforms?